examples/test_flow_dir/test_flow_dir.c

```diff
@@ -151,32 +160,102 @@ do_stats_display(struct rte_mbuf *pkt) {
         }
 }
 
+/*
+ * This function populates a set of flows to the sdn flow table.
+ * For each new flow a service chain is created and appended.
+ * Each service chain is of max 4 length. This function is not enabled by default.
+ */
+
+static void
+populate_ipv4_flow_table(void) {
```

Comment:
rename to `populate_sample_ipv4....` and add comment explaining that users can define their flow rules here if they want